fip-0026.md

```diff
@@ -339,251 +409,281 @@ This sets the marketplace configurations.
     "owner_public_key": "FIO6TJcqTGShkALUSdAyW4rhRt1kjgz5PuVUyf4pmaEEeb7VtCRy3",
     "comissionfee": "6",
     "listingfee": "3",
-    "duration": "90",
-    "warn_days": "10",
+    "e_break": 0,
```

Comment:
`e_break` not defined

Reply:
I'll remove it. I am defaulting it to 0 when the marketplace is configured anyway.